app/views/provider_interface/offer/checks/new.html.erb

```diff
@@ -0,0 +1,84 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_new_offer_path(@application_choice.id)) %>
+
+<%= render(FlashMessageComponent.new(flash: flash)) %>
+
+<%= form_with model: @wizard, url: provider_interface_application_choice_offers_path(@application_choice), method: :post do |f| %>
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-xl"><%= @application_choice.application_form.full_name %></span>
+    <%= t('.title') %>
+  </h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_error_summary %>
+      <div class="app-offer-panel app-banner app-banner--details">
+        <h2 class="govuk-heading-m">Course details</h2>
+        <%
+          rows = [
+          { key: 'Provider',
+            value: @wizard.course_option.provider.name_and_code,
+            action: 'Change',
+            change_path: nil,
+          },
+          { key: 'Course',
+            value: @wizard.course_option.course.name_and_code,
+            action: 'Change',
+            change_path: nil,
+          },
+          { key: 'Location',
+            value: @wizard.course_option.site.name_and_address,
+            action: 'Change',
+            change_path: nil,
+          },
+          { key: 'Full time or part time',
+            value: @wizard.course_option.study_mode.humanize,
+            action: 'Change',
+            change_path: nil,
+          }] %>
```

Comment:
Consider extracting outside of a view. Either moving to its own component or other clever solution.